tunnel_virtual_networks_test.go

```diff
@@ -0,0 +1,191 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTunnelVirtualNetworks(t *testing.T) {
+	setup()
+	defer teardown()
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodGet, r.Method, "Expected method 'GET', got %s", r.Method)
+		w.Header().Set("content-type", "application/json")
+		fmt.Fprintf(w, `{
+			"success": true,
+			"errors": [],
+			"messages": [],
+			"result": [
```

Comment:
can we please fix the JSON formatting? it is all over the place in this file (likely because it's a mix of Go and JSON)

Reply:
Tried to use 2 spaces. Hope this is what you meant. All files contains different JSON formatting. So I have no idea what to take as an example.